test/statistics.jl

```diff
@@ -96,6 +100,9 @@ end
 @test_approx_eq var([1], 1; mean=[2], corrected=false) [1.0]
 
 @test var(1:8) == 6.
+@test varm(1:8,1) == var(1:8)
```

Comment:
Put another way: if you assume that the distribution's true mean is a known constant, changing that constant should change your inferences about the distribution's unknown variance. This definition fails that basic restriction.